### ISSUE TITLE
Early stopping in the CLI?

### ISSUE THREAD

**Issue by adamyhe** [jmschrei/bpnet-lite#20]
The fit methods for BPNet and ChromBPNet have an optional early_stopping argument, which allow the models to end their training loops early if the validation loss plateaus. However, this argument hasn't been implemented in the command line tools. Is there a intentional reason for not including this?
